src/System.Text.Encodings.Web/tests/UnicodeHelpersTests.cs

```diff
@@ -21,17 +21,6 @@ public unsafe class UnicodeHelpersTests
         private static readonly UTF8Encoding _utf8EncodingThrowOnInvalidBytes = new UTF8Encoding(encoderShouldEmitUTF8Identifier: false, throwOnInvalidBytes: true);
 
         [Fact]
-        public void GetDefinedCharacterBitmap_ReturnsSingletonInstance()
```

Comment:
I don't see any other calls to GetDefinedCharacterBitmap in this file.  Are there tests for it somewhere else?